kinder/roadmap.md

```diff
@@ -0,0 +1,45 @@
+# Roadmap 🗺️
+
+This document outlines some goals, non-goals, and future aspirations for kinder.
+
+kinder is an example of [kind](https://github.com/kubernetes-sigs/kind) used as a library.
+
+All the kind commands will be available in kinder, side by side with additional commands designed
+for helping kubeadm contributors.
+
+High level goals for kinder v0.1 include:
+
+- [ ] Provide a local test environment for kubeadm development
+   - [x] Allow creation of nodes "ready for installing Kubernetes"
+   - [x] Provide pre built “developer” workflows for kubedam init, join, reset
+      - [x] init and init with phases
+      - [x] join and join with phases
+      - [x] init and join with automatic copy certs
+      - [x] Provide pre built “developer” workflow for kubeadm upgrades
+      - [x] reset
```

Comment:
huge +1 for doing this in `kinder` :upside_down_face:

Reply:
we should think about making kinder plugable in prow.
reset is definitely something we want to e2e eventually. the other day we fixed a panic that was uncaught, because we don't have signal. :\